.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,7 +2,14 @@
 Thank you for your Pull Request. Please provide a description above and review
 the requirements below.
 
-Bug fixes and new features should include tests and possibly benchmarks.
-
 Contributors guide: https://github.com/electron/electron/blob/master/CONTRIBUTING.md
--->
\ No newline at end of file
+-->
+
+##### Checklist
+<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
+
+- [ ] `npm test` passes
+- [ ] `npm run lint` passes
+- [ ] tests are included
```

Comment:
do we have links to guidelines for how to write tests?

Reply:
@nornagon i think we do, let me check